ansi-wl-pprint.cabal

```diff
@@ -1,7 +1,6 @@
 cabal-version:       >= 1.10
 name:                ansi-wl-pprint
-version:             0.6.9
-x-revision:          4
+version:             1.0.2
```

Comment:
Why pick version `1.0.2` instead of, say, `1.0.0`?

Reply:
It's the same version as https://hackage.haskell.org/package/prettyprinter-compat-ansi-wl-pprint